std/http/cookie.ts

```diff
@@ -1,13 +1,10 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 // Structured similarly to Go's cookie.go
 // https://github.com/golang/go/blob/master/src/net/http/cookie.go
-import { ServerRequest, Response } from "./server.ts";
 import { assert } from "../_util/assert.ts";
 import { toIMF } from "../datetime/mod.ts";
 
-export interface Cookies {
-  [key: string]: string;
-}
+export type Cookies = Record<string, string>;
```

Comment:
Shouldn't this be `Record<string, string | undefined>`?

With the domain / key being defined as `string` (as opposed to a literal type union), this type implies you'll get a value for every key which is obviously not the case, and the resulting header is typed as string where it could be undefined.

Reply:
If this is an issue, then the previous code behaved the same way.  It would be a seperate issue, with a reproduction of how it causes problems.